smartcontractkit/smart-contract-db
# feat: adds wallets page

# Overview

Adds wallets page

# Screenshots for Mobile and Desktop views (if applicable)

![image](https://user-images.githubusercontent.com/20880360/137029306-ce4b8657-97ec-4677-966f-3b7962639580.png)

### Manual Testing

Write the steps required for how might test this. Example:

1. Run `npm run dev`
2. In browser, goto `localhost:3000/`
3. Navigate to wallets page and verify data
4. Get results :tada:
